fstest/fstests/fstests.go

```diff
@@ -876,18 +877,18 @@ func Run(t *testing.T, opt *Opt) {
 			require.NoError(t, err)
 
 			pollInterval := make(chan time.Duration)
-			dirChanges := map[string]struct{}{}
```

Comment:
This change fixes some tests for the union backend, as multiple goroutines now write to these maps concurrently.